hero2vec.py

```diff
@@ -0,0 +1,162 @@
+#PyTorch libraries
```

Comment:
I can use some context here: is this `hero2vec` file ported over from the package? If so, should say so at the top of the file. If not, will do some review of the code.

Reply:
Oh, sorry. Should have been more specific about that. The hero2vec file uses the structure of an existing repo: https://github.com/ybw9000/hero2vec, but I modified the code to be fit with my own dataloader function and class. So it's probably 30-70, 70% based on the repo, and 30% I coded on my own.

Reply:
Got it, will do some review on this file. You should add this to the docstring at the top of the file though so a reader will have context on this file not being 100% your original work.

Reply:
Small teachable moment:

This is where more atomic commits would've been helpful. i.e.:
```
Commit 1 - You uploading the hero2vec.py module
Commit 2 - Change you made to hero2vec.py
Commit 3 - ...
```

Not a big deal now, but just something to note as a best practice in the future. Let's talk about how best to document your work and someone else's work at our next discussion.